webapp/store/views.py

```diff
@@ -384,10 +384,39 @@ def snap_details(snap_name):
             ):
                 is_users_snap = True
 
+        # build list of categories of the snap
+        # get all categories with their names
+        try:
```

Comment:
@tbille 

Because snap info API only returns 'slugs' of the categories (not full names) I need to call categories API as well and build a proper list of categories for the view.

Not sure if it's good to keep such logic here in view or better to move it out somewhere?

Reply:
Might be worth - at some point - doing something similar to the licenses? Categories won't change that often so we can store a static file on app launch. Every time we deploy it'll be updated.

Not suggesting for this PR, just an idea for future.